### ISSUE TITLE
Add 'more info' button for users to learn about "geo:"links

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/openstreetmap-website#2213]
In the **Share** panel, the user sees
```
Geo URI
geo:24.19410,120.86921?z=17
```
There should be a "?" link added at the side, for more information, as the user most probably has never heard about "geo:" links.

**Comment by HolgerJeromin** [comment]
No other part of osm.org has an integrated help.
I am not sure if this is really needed.

**Comment by tomhughes** [comment]
Indeed... I seem to recall I did point out that geo URIs were pretty obscure but people were very sure they wanted them. The main use case is mobile browsers I think, where they will often launch a mapping app.

**Comment by HolgerJeromin** [comment]
> Indeed... I seem to recall I did point out that geo URIs were pretty obscure

You did in #799

**Comment by jidanni** [comment]
Which would in fact be a good place for starters for the "?" info link to link to! Yup, "?" should link to #799!
